charts/argo-workflows/Chart.yaml

```diff
@@ -17,4 +17,4 @@ annotations:
     url: https://argoproj.github.io/argo-helm/pgp_keys.asc
   artifacthub.io/changes: |
     - kind: changed
-      description: Add support for sso filterGroupsRegex according to https://argoproj.github.io/argo-workflows/argo-server-sso/#filtering-groups
+      description: fix(argo-workflows): fix helm lint error when extraObjects is defined and same fix as one done for argo-cd #2116
```

Comment:
Hi @workwithprashant ,thank you for your PR.
Please fix [the failed lint](https://github.com/argoproj/argo-helm/actions/runs/7174398642/job/19536906907?pr=2384).
```
Package lint FAILED. 1 error(s) occurred:

  * invalid changes annotation. Please use quotes on strings that include any of the following characters: {}:[],&*#?|-<>=!%@

```

Otherwise LGTM. :)